Examples/S3EventNotifier/Package.swift

```diff
@@ -0,0 +1,52 @@
+// swift-tools-version: 6.0
+import PackageDescription
+
+// needed for CI to test the local version of the library
+import struct Foundation.URL
+
+let package = Package(
+    name: "CSVUploadAPINotificationLambda",
```

Comment:
I would use the same name as the directory : S3EventNotifier

Reply:
Oops, forgot to change this